hubblestack/modules/linux_sysctl.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""
+Module for viewing and modifying sysctl parameters
+"""
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import python libs
+import logging
+import os
+import re
+
+import salt.utils.systemd
+from hubblestack.utils.exceptions import CommandExecutionError
+import hubblestack.utils.data
+import hubblestack.utils.files
+import hubblestack.utils.stringutils
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = "sysctl"
+
+# TODO: Add unpersist() to remove either a sysctl or sysctl/value combo from
+# the config
+
+
+def __virtual__():
+    """
+    Only run on Linux systems
+    """
+    if __grains__["kernel"] != "Linux":
+        return (
+            False,
+            "The linux_sysctl execution module cannot be loaded: only available on Linux systems.",
+        )
+    return __virtualname__
+
+
+def default_config():
+    """
+    Linux hosts using systemd 207 or later ignore ``/etc/sysctl.conf`` and only
+    load from ``/etc/sysctl.d/*.conf``. This function will do the proper checks
+    and return a default config file which will be valid for the Minion. Hosts
+    running systemd >= 207 will use ``/etc/sysctl.d/99-salt.conf``.
+    CLI Example:
+    .. code-block:: bash
+        salt -G 'kernel:Linux' sysctl.default_config
+    """
+    if (
+        salt.utils.systemd.booted(__context__)
```

Comment:
salt.utils.systemd is already ported to hubblestack.utils.systemd and it has these two methods as well:
booted, version
Please use that.